### DISCUSSION TITLE
Filter order table by coupon code

### DISCUSSION THREAD

**Discussion by philipboomy** [craftcms/commerce#3776] [Ideas]
It would be nice if we could use the built in filters to filter orders with a specific coupon code.

Right now we only got these seen in the screenshot.

![Screenshot 2024-11-18 at 11 11 58](https://github.com/user-attachments/assets/889afc24-832f-4d5e-a093-909293871ebd)

**Comment by nfourtythree** [comment]
Commerce 5.3 has now been released with the ability to filter by coupon code included.

Thanks!

**Reply by terryupton** [reply]
Could this be ported back to commerce 4? I have a client with this need but aren't ready to upgrade to 5 yet. 
@nfourtythree
